features/steps/events_steps.rb

```diff
@@ -362,7 +362,7 @@ def icecube_schedule_to_time_restriction(sched, time_zone)
 
 Given /^user (\S+) has the following notification rules:$/ do |contact_id, rules|
   contact = Flapjack::Data::Contact.find_by_id(contact_id, :redis => @redis)
-  timezone = contact.timezone
+  time_zone = contact.time_zone
```

Comment:
why are we s/timezone/time_zone/ ? the attribute is 'timezone' in the api http://flapjack.io/docs/1.0/jsonapi/#contacts so it'd be nice to be consistent unless there's a good reason